CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
```

Comment:
shall we mention the previous API with mapView as argument is deprecated?

Reply:
was thinking about it, but thought about adding this level of details to release notes, WDYT?